packages/d3fc-chart/src/cartesian.js

```diff
@@ -58,8 +59,16 @@ export default (...args) => {
                 .attr('class', d => `x-label ${d}-label`)
                 .text(xLabel(data));
 
-            yLabelDataJoin(container, [yOrient(data)])
-                .attr('class', d => `y-label ${d}-label`)
+            const yOrientValue = yOrient(data);
+            const yLabelContainer = yContainerDataJoin(container, [yOrientValue])
+                .attr('class', d => `y-label-container ${d}-label`)
+                .style('grid-column', yOrientValue === 'left' ? 1 : 5)
```

Comment:
I think these grid-column styles shouldn't be needed - the css already has these for the `.left-label` and `.right-label` classes.